meson.build

```diff
@@ -0,0 +1,25 @@
+project('foo', 'cython', 'cpp')
+
+py = import('python').find_installation()
+dep_py = py.dependency()
+
+harfbuzz_includes = include_directories('harfbuzz/src')
+harfbuzz = static_library(
+    'harfbuzz',
+    'harfbuzz/src/harfbuzz-subset.cc',
+    'harfbuzz/src/hb-coretext.cc',
+    'harfbuzz/src/hb-directwrite.cc',
+    'harfbuzz/src/hb-uniscribe.cc',
+    cpp_args: '-DHB_NO_MT=1 -DHB_EXPERIMENTAL_API=1',
+    include_directories: harfbuzz_includes,
+)
```

Comment:
If you are going to use meson, why not use harfbuzz as a subproject (or simply as a [dependency](https://mesonbuild.com/Reference-manual_functions.html#dependency), so that #45 could be fixed as well). You can

Reply:
Yes, that's the idea, but first I want to get it to actually compile. It still complains about missing hb.h.